x-pack/plugins/code/server/routes/repository.ts

```diff
@@ -78,8 +80,9 @@ export function repositoryRoute(
   });
 
   // Remove a git repository
-  server.route({
+  server.securedRoute({
```

Comment:
leave a comment as a reminder here to explain what does this `secureRoute` do a bit in today's sync.